### PR TITLE
`dex` - added TVL to new DEX

### PR DESCRIPTION
**Pull request by thedoublejay**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx [hunk 1]
```diff
@@ -97,6 +102,33 @@ export function DexScreen (): JSX.Element {
...
+      >
+        <View style={tailwind('flex flex-col')}>
+          <ThemedText light={tailwind('text-gray-500')} dark={tailwind('text-gray-500')} style={tailwind('text-xs')}>{translate('screens/DexScreen', 'Total Value Locked (USD)')}</ThemedText>
+          <ThemedText>
+            <NumberFormat
+              displayType='text'
+              prefix='$'
+              renderText={(val: string) => (
+                <ThemedText
+                  dark={tailwind('text-gray-400')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx]: I find it less obvious in dark mode, compared to how contrast it is in light mode. Can we go with 
```suggestion
                  dark={tailwind('text-gray-50')}
```
IMG_D25A8F7BBC71-1

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx [hunk 2]
```diff
@@ -97,6 +102,33 @@ export function DexScreen (): JSX.Element {
 
   return (
     <>
+      <ThemedView
+        dark={tailwind('bg-gray-800 border-b border-gray-700')}
+        light={tailwind('bg-white border-b border-gray-200')}
+        style={tailwind('flex flex-row px-4 py-3')}
+      >
+        <View style={tailwind('flex flex-col')}>
+          <ThemedText light={tailwind('text-gray-500')} dark={tailwind('text-gray-500')} style={tailwind('text-xs')}>{translate('screens/DexScreen', 'Total Value Locked (USD)')}</ThemedText>
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:111]: same label color for both mode 👀?

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx [hunk 3]
```diff
@@ -97,6 +102,33 @@ export function DexScreen (): JSX.Element {
...
+          <ThemedText>
+            <NumberFormat
+              displayType='text'
+              prefix='$'
+              renderText={(val: string) => (
+                <ThemedText
+                  dark={tailwind('text-gray-200')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:118]: by our color convention, dark mode for `text-gray-900` should go `text-gray-50`

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx (hunk 2 above)
Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:111]: Yes it's based on Section title

**Review by kyleleow — APPROVED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx (hunk 2 above)
Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:111]: ah damn it's only for default, others used -400

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:111]: Let me update

**Pull request merged**